### ISSUE TITLE
Kotlinjs support

### ISSUE THREAD

**Issue by MTRNord** [spdx/spdx-gradle-plugin#129]
Hi :) I tried to use this in a kotlinjs project but I only got this error:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':ui'.
> Could not create task ':ui:spdxSbomForRelease'.
   > Configuration with name 'runtimeClasspath' not found.
```

from another issue I saw that it might make sense to look at the dependencies task which gives this output:

```
❯ ./gradlew --console plain :ui:dependencies | fgrep ' - '
fgrep: warning: fgrep is obsolescent; using grep -F
jsApiElements - API elements for main. (n)
jsCompilationApi - API dependencies for js/main (n)
jsCompilationCompileOnly - Compile only dependencies for js/main. (n)
jsCompilationImplementation - Implementation only dependencies for js/main. (n)
jsCompilationRuntimeOnly - Runtime only dependencies for js/main. (n)
jsCompileClasspath - Compile classpath for js/main.
jsNpmAggregated - NPM configuration for compilation 'main' (target js (js)).
jsRuntimeClasspath - Runtime classpath of js/main.
jsRuntimeElements - Elements of runtime for main. (n)
jsSourcesElements - Source files of main compilation of js. (n)
jsTestCompilationApi - API dependencies for js/test (n)
jsTestCompilationCompileOnly - Compile only dependencies for js/test. (n)
jsTestCompilationImplementation - Implementation only dependencies for js/test. (n)
jsTestCompilationRuntimeOnly - Runtime only dependencies for js/test. (n)
jsTestCompileClasspath - Compile classpath for js/test.
jsTestNpmAggregated - NPM configuration for compilation 'test' (target js (js)).
jsTestRuntimeClasspath - Runtime classpath of js/test.
kotlinCompilerPluginClasspathJsMain - Kotlin compiler plugins for compilation
kotlinCompilerPluginClasspathJsTest - Kotlin compiler plugins for compilation
kotlinCompilerPluginClasspathMetadataMain - Kotlin compiler plugins for compilation
ktlint - Main ktlint-gradle configuration
ktlintBaselineReporter - Provides KtLint baseline reporter required to generate baseline file
ktlintReporter - All ktlint custom reporters dependencies
ktlintRuleset - All ktlint rulesets dependencies
metadataApiElements - API elements for main. (n)
metadataCompilationApi - API dependencies for metadata/main (n)
metadataCompilationCompileOnly - Compile only dependencies for metadata/main. (n)
metadataCompilationImplementation - Implementation only dependencies for metadata/main. (n)
metadataCompilationRuntimeOnly - Runtime only dependencies for metadata/main. (n)
metadataCompileClasspath - Compile classpath for metadata/main.
metadataSourcesElements - Source files of main compilation of metadata. (n)
(c) - A dependency constraint, not a dependency. The dependency affected by the constraint occurs elsewhere in the tree.
(*) - Indicates repeated occurrences of a transitive dependency subtree. Gradle expands transitive dependency subtrees only once per project; repeat occurrences only display the root of the subtree, followed by this annotation.
(n) - A dependency or dependency configuration that cannot be resolved.
```

this seems to indicate that its `jsRuntimeClasspath` rather than `runtimeClasspath` in kotlinjs projects. It would be awesome if this could be supported

**Comment by loosebazooka** [comment]
`runtimeClasspath` is the default configuration (since that works for most java projects)

You probably just need to configure it -- https://github.com/spdx/spdx-gradle-plugin?tab=readme-ov-file#configuration using the configurations parameter

**Comment by MTRNord** [comment]
Oh I missed that fully. I will give that a try thanks :)

**Comment by MTRNord** [comment]
`configurations.set(listOf("jsRuntimeClasspath"))` indeed seems to do the trick here
